### PR TITLE
👷 Add basic testing against Ansible 2.8

### PR DESCRIPTION
**Pull request by webknjaz**
#### PR Type

- Testing Pull Request

### CODE REVIEW

**Comment by decentral1se** [comment]
Related: https://github.com/ansible/molecule/issues/1727.

**Comment by webknjaz** [comment]
FTR: I've triggered the full test run in my fork.
It will actually run those hidden jobs with Ansible 2.8b1: https://travis-ci.com/webknjaz/molecule/builds/108597824

**Comment by webknjaz** [comment]
@decentral1se jobs are failing as expected but they don't affect the overall build status and only fire on cron or manual trigger for now. I believe this is ready and we'll be able to follow-up and remove "allowed failures" mark from them once tests are fixed.

**Comment by webknjaz** [comment]
P.S. anyone setting approved status next — feel free to merge immediately.

File: tox.ini [hunk 1]
```diff
@@ -44,6 +44,7 @@ extras =
     vagrant
     windows
 commands_pre =
+    ansible28: pip install --pre ansible==2.8b1
```
Comment (ssbarnea) [inline tox.ini]: Replace it with `ansible>=2.8b1,<2.9` and it will be ok, as we do not want to bump it on each new pre-release.

Comment (ssbarnea) [reply tox.ini]: I also hope that final merge does not contain emoji. I don't mind them on Github, but I do not want to see them in commit messages.

Comment (webknjaz) [reply tox.ini]: FTR https://github.blog/2018-10-16-future-of-software/#suggested-changes-public-beta

**Comment by webknjaz** [comment]
@gundalow it's the best you can do with tox now

**Comment by ssbarnea** [comment]
@gundalow Please be more careful not to allow emoji into commit messages. That one already went in and we will never be able to get it out. I don't want to see git log looking like the xmas tree.

**Pull request merged**